options/locale/locale_en-US.ini

```diff
@@ -1369,7 +1369,7 @@ issues.context.reference_issue = Reference in New Issue
 issues.context.edit = Edit
 issues.context.delete = Delete
 issues.no_content = There is no content yet.
-issues.close_issue = Close
+issues.close_issue = Close issue
```

Comment:
I agree, but what should we do with all these other instances?
![image](https://user-images.githubusercontent.com/20454870/225549999-89f7141d-6199-4700-8a2c-0f136e105221.png)

Reply:
> ... but what should we do with all these other instances?

I think we can just leave them, they will be fixed on Crowdin later.

And before that, I don't think it would be a problem for the text to fallback to English as "Close Issue".

Reply:
Done